input/resources/au-medication.xml

```diff
@@ -32,61 +32,56 @@
+      <comment value="Australian coding are typical medicine/product concept codes.&#xD;&#xA;&#xD;&#xA;A code (or set of codes) that specify this medication, or a textual description if no code is available. Usage note: This could be a standard medication code such as a code from RxNorm, SNOMED CT, IDMP etc. It could also be a national or local formulary code, optionally with translations to other code systems."/>
       <binding>
-        <strength value="preferred"/>
-        <valueSet value="http://terminology.hl7.org.au/ValueSet/pbs-item"/>
-      </binding>
-    </element>
-    <element id="Medication.code.coding:pbs.system">
-      <path value="Medication.code.coding.system"/>
-      <min value="1"/>
-      <fixedUri value="http://pbs.gov.au/code/item"/>
-    </element>
-    <element id="Medication.code.coding:gtin">
-      <path value="Medication.code.coding"/>
-      <sliceName value="gtin"/>
-      <short value="Medication Package Global Trade Item Number (GTIN)"/>
-      <comment value="GTIN value from http://www.gs1.org/gtin."/>
-      <binding>
-        <strength value="preferred"/>
-        <valueSet value="http://terminology.hl7.org/ValueSet/v3-GTIN"/>
+        <extension url="http://hl7.org/fhir/tools/StructureDefinition/additional-binding">
+          <extension url="purpose">
+            <valueCode value="preferred"/>
+          </extension>
+          <extension url="valueSet">
+            <valueCanonical value="https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1"/>
+          </extension>
+          <extension url="documentation">
+            <valueMarkdown value="This value set is for use when AMT (Australian Medicines Terminology) codes are preferred."/>
+          </extension>
+        </extension>
+        <extension url="http://hl7.org/fhir/tools/StructureDefinition/additional-binding">
+          <extension url="purpose">
+            <valueCode value="preferred"/>
+          </extension>
+          <extension url="valueSet">
+            <valueCanonical value="http://terminology.hl7.org.au/ValueSet/pbs-item"/>
+          </extension>
+          <extension url="documentation">
+            <valueMarkdown value="This value set is for use when PBS (Pharmaceutical Benefits Scheme) item codes are preferred."/>
+          </extension>
+        </extension>
+        <extension url="http://hl7.org/fhir/tools/StructureDefinition/additional-binding">
+          <extension url="purpose">
+            <valueCode value="preferred"/>
+          </extension>
+          <extension url="valueSet">
+            <valueCanonical value="http://terminology.hl7.org.au/ValueSet/mims"/>
+          </extension>
+          <extension url="documentation">
+            <valueMarkdown value="This value set is for use when MIMS codes are preferred."/>
```

Comment:
Flagging for consideration: in the table AMT, PBS and GTIN acronyms are spelled out in full, should MIMS be too? Noting that it is rarely seen in full form.

Reply:
What is MIMS an abbreviation for?

Reply:
Suggest amending to "This value set is for use when MIMS (Monthly Index of Medical Specialties) codes are preferred."

Reply:
fixed.